src/System.Runtime.WindowsRuntime/src/System/Threading/WindowsRuntimeSynchronizationContext.cs

```diff
@@ -63,7 +63,8 @@ internal sealed class WinRTSynchronizationContextFactory : WinRTSynchronizationC
         private static readonly ConditionalWeakTable<IDispatcherQueue, WinRTDispatcherQueueBasedSynchronizationContext> s_dispatcherQueueContextCache =
             new ConditionalWeakTable<IDispatcherQueue, WinRTDispatcherQueueBasedSynchronizationContext>();
 
-        public override SynchronizationContext Create(object dispatcherObj)
+        // System.Private.Corelib will call WinRTSynchronizationContextFactory.Create() via reflection
+        public static SynchronizationContext Create(object dispatcherObj)
```

Comment:
Can you check that the method is actually included in the release builds? We run ILLinker on corefx release builds to delete dead/unused code. You may need to add `ILLinkTrim.xml` to keep it.

Reply:
the method exists even in release build.